backend/app/src/users/users.controller.ts

```diff
@@ -1,37 +1,24 @@
 import { Controller, Delete, Get, Param, Logger } from '@nestjs/common';
 import { UsersService } from './users.service';
-const logger = new Logger('MyLogger');
+// const logger = new Logger('MyLogger');
```

Comment:
Don't forget to delete this commented out code and it's import on line 1

Reply:
I was thinking of leaving this line as we may find it useful when we need to log issues rather than using console.log

Reply:
Done